src/tests/system/tests/test_id.py

```diff
@@ -0,0 +1,455 @@
+""" SSSD Client identification
+
+:requirement: IDM-SSSD-REQ: Client side performance improvements
+"""
+
+from __future__ import annotations
+
+import pytest
+from sssd_test_framework.roles.client import Client
+from sssd_test_framework.roles.generic import GenericProvider
+from sssd_test_framework.topology import KnownTopologyGroup
+
+
+@pytest.mark.topology(KnownTopologyGroup.AnyProvider)
+def test_id__getpwnam(client: Client, provider: GenericProvider):
+    """
+    :title: Resolving user with getpwnam by id(name)
```

Comment:
Remove -ing form in all titles.

Change to: `Resolve user by name with id`, similarly on other places.